sql/parser/sql.y

```diff
@@ -2774,14 +2774,6 @@ func_expr_windowless:
 // Special expressions that are considered to be functions.
 func_expr_common_subexpr:
   COLLATION FOR '(' a_expr ')' { unimplemented() }
-| CURRENT_DATE
```

Comment:
I think you should revert this. The variable form of `CURRENT_{DATE,TIMESTAMP}` is part of the SQL standard.